nexus/types/src/deployment/execution/utils.rs

```diff
@@ -85,3 +85,19 @@ pub fn blueprint_nexus_external_ips(blueprint: &Blueprint) -> Vec<IpAddr> {
         })
         .collect()
 }
+
+/// Return the addresses on which this blueprint's external DNS servers listen
+/// for DNS queries.
+pub fn blueprint_external_dns_resolver_ips(
```

Comment:
```suggestion
pub fn blueprint_external_dns_nameserver_ips(
```

(a lot of stuff uses these interchangeably but I believe resolvers are *client*-side components that initiate DNS queries and I find it useful to keep this distinction)

Reply:
you're totally right, yes. i'd just picked a poor name here, i know better :(

Reply:
lol, no sweat!  Honestly, I see these used interchangeably more often than I see them used distinctly.